content/organization/events/speakers/index.en.md

```diff
@@ -27,6 +27,47 @@ weight: 53
 
 * Remember the [rules about Prioritization of Underrepresented/Minority Genders](/about/mission/#r-ladies-rules--guidelines).
 
+* In line with R-Ladies’ Mission Statement and with our [Black Lives Matter Statement](https://blog.rladies.org/post/2020-06-06-blm/) 
+to achieve participation and representation in our events of generally excluded groups, taking into account the intersectionality of gender, race, language and geography; for these reasons we request:
+
+  - In chapter events, invite speakers of diverse race, language and geography (i.e., avoid a one year period with all white speakers at the chapter events).
+  - For panel discussions of 3 or more panelists, invite speakers of diverse race, language and geography (i.e., avoid all white panels).
+
+### How to find speakers? 
+- You can use the [R-Ladies Directory](https://rladies.org/directory/) to identify potential speakers that are not: white, native anglophone, or from high income level countries.
+- You can reach other communities to request for speakers information (see lists below).
+- You can reach other R-Ladies in the R-Ladies Community Slack.
+- You can use others directories to look for speakers (see list below). 
```

Comment:
```suggestion
- You can follow other communities on social media to get to know their members and potential speakers.
```

re removal: that would already be covered by line 38 and in that form, checking the directories would be listed higher up

re follow other communities: is this either too obvious or too vague?

Reply:
Obvious: it doesn't make any damage to explicitly say this.